src/server/scripts/Commands/cs_npc.cpp

```diff
@@ -1533,6 +1534,44 @@ class npc_commandscript : public CommandScript
         return true;
     }
 
+	static bool HandleNpcReloadCommand(ChatHandler* handler, char const* args)
+	{
+		Creature* creatureTarget = handler->getSelectedCreature();
+		if (!creatureTarget || creatureTarget->IsPet() || creatureTarget->IsGuardian())
+		{
+			handler->PSendSysMessage(LANG_SELECT_CREATURE);
+			handler->SetSentErrorMessage(true);
+			return false;
+		}
+
+		creatureTarget->ReLoad(false);
+
+		char* all_str = args ? strtok((char*)args, " ") : nullptr;
```

Comment:
mby use Args helper?

Reply:
Not needed here, Args helper do not support optional args yet